sample-app/src/components/VerticalResults.tsx

```diff
@@ -17,13 +27,16 @@ interface VerticalResultsDisplayProps {
  *                to be used.
  */
 export function VerticalResultsDisplay(props: VerticalResultsDisplayProps): JSX.Element | null {
-  const { CardComponent, results, cardConfig = {}, isLoading = false } = props;
+  const { CardComponent, results, cardConfig = {}, isLoading = false, customCssClasses, compositionMethod } = props;
+  const cssClasses = useComposedCssClasses(builtInCssClasses, customCssClasses, compositionMethod);
 
   if (results.length === 0) {
     return null;
   }
 
-  const resultsClassNames = classNames('VerticalResults', { 'VerticalResults--loading': isLoading });
+  const resultsClassNames = cssClasses.results___loading
+    ? classNames({ [cssClasses.results___loading]: isLoading })
+    : '';
```

Comment:
Are we able to get rid of the `VerticalResults` CSS class because it has no associated styling?

Reply:
Yes, but I could leave it in so that it's customizable

Reply:
I'm okay getting rid of it! Trim the fat. Drain the swamp.